examples/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/ipfs/go-datastore v0.6.0
 	github.com/ipfs/go-log/v2 v2.5.1
-	github.com/libp2p/go-libp2p v0.24.0
+	github.com/libp2p/go-libp2p v0.24.2
```

Comment:
I don't think this dependency update is needed, is it?

Reply:
not really but way not ?

Reply:
PRs should be as minimal as possible. If it's not required to update a dependency, it's required to not update a dependency.

Reply:
@marten-seemann  Done